packages/react/src/components/MultiSelect/MultiSelect.js

```diff
@@ -85,6 +86,7 @@ const MultiSelect = React.forwardRef(function MultiSelect(
     disabled,
     initialSelectedItems,
     onChange,
+    controlledItems: value,
```

Comment:
selected
```suggestion
    selectedItems
```

Reply:
> 

I have a comment here, I can use the name selectedItems on props but there is one property on the same scope, I don't know if just rename it on destructucting `selectedItems: selected` or rename the other selectedItems variable, is on line 97.

Also on useSelection can be a little confusing because there is a property called selectedItems